docs/whatsnew/5.3.rst

```diff
@@ -181,6 +182,22 @@ we can take the sum of ND masked quantities along the ``1`` axis like so::
      array([ True, False]),
      StdDevUncertainty([1.41421356, 1.73205081]))
 
+.. _whatsnew-5.3-coordinates-refresh-site-registry:
+
+Refresh cached site registry for ``coordinates.EarthLocation`` methods
+======================================================================
```

Comment:
Differently from the change log entries a What's New entry should contain more links rather than less:
```suggestion
Refresh cached site registry for |EarthLocation| methods
========================================================
```